src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -231,12 +231,12 @@ LRESULT IslandWindow::_OnSizing(const WPARAM wParam, const LPARAM lParam)
         // If user has dragged anything but the top or bottom border (so e.g. left border,
         // top-right corner etc.), then this means that the width has changed. We thus ask to
         // adjust this new width so that terminal(s) is/are aligned to their character grid(s).
-        clientWidth = gsl::narrow_cast<int>(_pfnSnapDimensionCallback(true, gsl::narrow_cast<float>(clientWidth)));
+        clientWidth = static_cast<decltype(clientWidth)>(_pfnSnapDimensionCallback(true, static_cast<float>(clientWidth)));

```

Comment:
You're right, I'll revert it to use `gsl::narrow_cast`, but I'll keep the `decltype(clientWidth)`, because that adds something. `clientWidth` is a `long` and not a `int` after all.